projects/gigya/src/config/default-gigya-config.ts

```diff
@@ -0,0 +1,11 @@
+import { GigyaConfig } from './gigya-config';
+
+export const DEFAULT_GIGYA_CONFIG: GigyaConfig = {
```

Comment:
Could we dropped the default config?

Let's enforce user to provide the config for his use case. Default config doesn't give enough value.

Reply:
Removed the default config.